health_monitor/health_monitor.ino

```diff
@@ -0,0 +1,167 @@
+#include <BLEDevice.h>
+#include <string>
+
+#define uS_TO_S_FACTOR 1000000 /* Conversion factor for micro seconds to seconds */
+#define TIME_TO_SLEEP 5        /* Time ESP32 will go to sleep (in seconds) */
+#define RSSI_LOWER_BOUND -100  // rssi less than this ignored
+#define SCAN_SECONDS 3
+
+#define PREFIX "Health Monitor - "
+#define CAT    "Zombie -1"
+#define IMMUNE "Immune"
+#define SUPER  "Super Healthy"
+#define HEALTY "Healthy"
+#define A_SYMPTOMATIC "Asymptomatic"
+#define SICK "Sick"
+#define ZOMBIE "Zombie"
+
```

Comment:
FWIW I think the last time I used the arduino blend of C++, `constexpr` was available.

Reply:
Indeed it seems to be